docs/reference/settings/ssl-settings.asciidoc

```diff
@@ -26,12 +26,14 @@ Defaults to +{client-auth-default}+.
 endif::client-auth-default[]
 endif::server[]
 
-ifdef::verifies[]
+ifdef::server[]
 +{ssl-prefix}.ssl.verification_mode+::
 (<<static-cluster-setting,Static>>)
-Controls the verification of certificates.
+Defines how to verify the certificates presented by another party in the TLS 
+connection. This setting accepts the following values for TLS _clients_, but 
+should not be set for TLS _servers_:
```

Comment:
The content is 
> accepts the following values for TLS _clients_, but should not be set for TLS _servers_

But the condition check is
> ifdef:server[]

I think they contradcit to each other: "If it is a server definition, show this setting that is for client only"?

Watcher settings in `notification-settings.asciidoc` have `:server!:`. With the proposed change, this setting will no longer show up for watchers. I think it's wrong because watchers are TLS clients.

Also, if we no longer need `verifies`, it would be great to remove them from other places.

Reply:
You're right, this check isn't needed here. I'm wondering if it's needed elsewhere. In [`security-settings.asciidoc`,](https://github.com/elastic/elasticsearch/blame/main/docs/reference/settings/security-settings.asciidoc#L2437) the definition for `:server:` is included for both the HTTP and Transport settings, but has no value in either case. So every field without an `ifdef` statement is included, plus any field where `ifdef:server[]` is defined -- there's no exclusion happening, so it seems like defining `:server:` isn't necessary.